librocksdb_sys/crocksdb/c.cc

```diff
@@ -3779,4 +3798,109 @@ int crocksdb_keyversions_type(const crocksdb_keyversions_t *kvs, int index) {
   return kvs->rep[index].type;
 }
 
+struct SstFileSeqNoModifier {
```

Comment:
Better to use a more general name like `ExternalSstFileModifier`.

Reply:
I prefer SstFileModifer like SstFileReader and SstFileWriter